tools/sapp/sapp/interactive.py

```diff
@@ -453,6 +457,14 @@ def issues(
             codes: int or list[int]        issue codes to filter on
             callables: str or list[str]    callables to filter on (supports wildcards)
             filenames: str or list[str]    filenames to filter on (supports wildcards)
+            exact_trace_length_to_sources: 
+                int or list[int]    min trace length to sources to filter on 
```

Comment:
Might be worth updating the description to mirror the naming. Something like:
```
exact values for min trace length to {sources,sinks} to filter on 
```
and 
```
maximum value for min trace length to {sources,sinks} to filter on 
```